examples/2d/elixir_advection_basic.jl

```diff
@@ -45,27 +36,17 @@ summary_callback = SummaryCallback()
 stepsize_callback = StepsizeCallback(cfl=1.6)
 
 save_solution = SaveSolutionCallback(interval=100,
-                                     save_initial_solution=true,
-                                     save_final_solution=true,
-                                     solution_variables=:conservative)
-
-save_restart = SaveRestartCallback(interval=100,
-                                   save_final_restart=true)
+                                     solution_variables=:primitive)
 
-analysis_interval = 100
-alive_callback = AliveCallback(analysis_interval=analysis_interval)
```

Comment:
Why don't you want to keep the `AliveCallback`? I don't insist on using it by it would be okay for me to keep it.

Reply:
Well, essentially the same reason I killed the `SaveRestartCallback` - to cut down on lines of code and to reduce the clutter. I thought - if this is the first elixir a user is ever going to see, it should contain the least amount of "concepts" without being useless. Since the alive interval is the definition of non-essential, I cut it at least from this elixir.

Reply:
Okay, sounds reasonable.

Reply:
OK, I'll leave it as it is then.

Reply:
I mean, as it is in the current PR.